geopandas/io/tests/test_sql.py

```diff
@@ -359,6 +369,27 @@ def test_write_postgis_default(self, engine_postgis, df_nybb):
         df = read_postgis(sql, engine, geom_col="geometry")
         validate_boro_df(df)
 
+    def test_write_postgis_geography(self, engine_postgis, df_nybb):
+        """Tests that GeoDataFrame can be written to PostGIS with geography type."""
+        engine = engine_postgis
+        table = "nybb"
+
+        # If table exists, delete it before trying to write with defaults
+        drop_table_if_exists(engine, table)
+
+        # Write to db
+        write_postgis(
+            df_nybb.to_crs("epsg:4326"),
+            con=engine,
+            name=table,
+            if_exists="fail",
+            use_geography=True,
+        )
+        # Validate
+        sql = text("SELECT * FROM {table};".format(table=table))
+        df = read_postgis(sql, engine, geom_col="geometry")
```

Comment:
Could we validate here that we actually created a geography type column? (for example, querying `SELECT * FROM geography_columns;` and checking that the column is present (and not in `geometry_columns`))